addons/web_kanban/static/src/js/kanban_view.js

```diff
@@ -510,12 +510,17 @@ var KanbanView = View.extend({
             var dataset = new data.DataSetSearch(self, rel_name, self.dataset.get_context(rel.context));
             dataset.name_get(_.uniq(rel.ids)).done(function(result) {
                 result.forEach(function(nameget) {
-                    // white color (0) should not be selected, 1 instead
-                    var color = KanbanRecord.prototype.kanban_getcolor(nameget[1]) + 1;
-                    var $tag = $('<span>')
-                        .addClass('o_tag oe_kanban_color_' + color)
-                        .attr('title', _.str.escapeHTML(nameget[1]));
-                    $(rel.elements[nameget[0]]).append($tag);
+                    var $element = $(rel.elements[nameget[0]]);
+                    if ($element.hasClass('o_m2m_tags')) {
```

Comment:
@msh-odoo we cannot rely on class name, there is another better way, define * options={"m2m_tags": True}* in field where define in kanban, not where we are showing, and add this.m2m_option as a dictionary and add those information in that, and change according to that rest of code.